wiki/History_of_osu!/2015/en.md

```diff
@@ -1,39 +1,185 @@
+
+The [osu!news YouTube Channel](https://www.youtube.com/channel/UCZKQIqv9O2tddMNUMAxWaqQ "YouTube") uploaded its [first video](https://www.youtube.com/watch?v=KQbudVxEjr8 "YouTube") to the public on 8 May 2015. The channel was run by various members of the community and aimed to create videos on skin showcases, tournaments, mapping, contests, and other news from the community [[12,13]][r].
 
-Links:
+The osu!keyboard (also known as the "osu! 'nono' keyboard") was added to the osu!store on 30 May 2015, retailing for about $40 at launch. Initially announced in a [tweet by peppy](https://twitter.com/ppy/status/603797988742336512) on 28 May 2015, there were only 100 keyboards available at launch, and were more akin to a keypad than a full-sized keyboard [[14,15]][r].
 
-- [Comp 3 6c](https://www.youtube.com/watch?v=lrWyqpNUwBo)
+![](img/osu-keyboard.jpg "Image of the osu! keyboard teased in the tweet by peppy")
+
+## June
+
+Implementation of the username change system in the osu!store was set live on 16 June 2015, although full functionality of the system was not available until 18 June 2015. The changes made it so that players must pay $4 USD for their first name change, $8 USD for their second, $16 USD for their third, after which the price would double with each succeeding purchase with a cap at $100 USD; however, if a user had bought an [osu!supporter tag](https://osu.ppy.sh/home/support), then their first change would be free [[16,17,18,19]][r].
+
+The ability to see what a replay would look like without [mods](/wiki/Game_modifier) was added on 23 June 2015 as per a [feature request](https://osu.ppy.sh/community/forums/topics/97560), although the change only made the mods [Double Time](/wiki/Game_modifier/Double_Time), [Half Time](/wiki/Game_modifier/Half_Time), and [Flashlight](/wiki/Game_modifier/Flashlight) toggle-able [[20]][r].
 
 ## July
 
-peppy and flyte briefly explained why the temporary osu!web redesign subdomain was `jizz.ppy.sh` in an episode of osu!talk.
+On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test cutting-edge features, contribute graphical/sound resources, or get help working with osu! services could come and join the team and talk to the osu! team directly [[21]][r].
```

Comment:
```suggestion
On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test cutting-edge features, contribute graphical/sound resources, or get help working with osu! services could come and join the team and talk to the osu! team directly [[21]][r].
```
```suggestion
On 15 July 2015, it was announced that a new public [Slack](https://slack.com/ "Slack") team was created for the purpose of better communicating with public contributors. [peppy](https://osu.ppy.sh/users/2) initially made the announcement on his blog and had explained that anyone who was willing to help test new cutting-edge features, contribute to graphical/sound resources, or get help working with osu! services could come to join the team and talk to the osu! team directly [[21]][r].
```

Reply:
the writing here was intentional; "to" and "new" change the meaning